tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -40,6 +40,10 @@ library
   default-language:    Haskell2010
   include-dirs: .
 
+custom-setup
+  setup-depends: Cabal
+                , proto-lens-protoc == 0.2.*
```

Comment:
Are you able to remove `build-depends: proto-lens-protoc` from the library above?